### ISSUE TITLE
running single with GRCh38

### ISSUE THREAD

**Issue by jjfarrell** [statgen/EPACTS#29]
Does EPACTS work with GRCh38 aligned VCFs? I tried using the --ref option and the scripts generated are still using GRCh37 coordinates?  Is there a workaround?

epacts single   --vcf manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz --ped ../kage_w_head_model1.ped --min-maf 0.01 --min-mac 10  --pheno AD --test b.wald --pass --ref /restricted/projectnb/casa/ref/GRCh38_full_analysis_set_plus_decoy_hla.fa --out test     Detected phenotypes with 2 unique values - 0 and 1 - considering them as binary phenotypes... re-encoding them into 1 and 2
Successfully written phenotypes and 0 covariates across 2000 individuals
Processing chromosome 1...
Processing chromosome 2...
Processing chromosome 3...
Processing chromosome 4...
Processing chromosome 5...
Processing chromosome 6...
Processing chromosome 7...
Processing chromosome 8...
Processing chromosome 9...
Processing chromosome 10...
Processing chromosome 11...
Processing chromosome 12...
Processing chromosome 13...
Processing chromosome 14...
Processing chromosome 15...
Processing chromosome 16...
Processing chromosome 17...
Processing chromosome 18...
Processing chromosome 19...
Processing chromosome 20...
Processing chromosome 21...
Processing chromosome 22...
Processing chromosome X...
Processing chromosome Y...
Processing chromosome MT...

When the makefile is run, it does not find the chr1-chr22 regions as it is looking for 1-22,X,Y,MT.

       make -f /rprojectnb2/kageproj/manta/results.epacts/test.Makefile -j [# of parallel jobs]
Or perform sanity checking using the following command:
        make -f /rprojectnb2/kageproj/manta/results.epacts/test.Makefile -n
[farrell@scc-hadoop results.epacts]$ make -f /rprojectnb2/kageproj/manta/results.epacts/test.Makefile -j  5
Rscript /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install/share/EPACTS/epactsSingle.R --vanilla /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install /rprojectnb2/kageproj/manta/results.epacts/test.phe NULL /rprojectnb2/kageproj/manta/results.epacts/test.ind /rprojectnb2/kageproj/manta/results.epacts/manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz 11:10000001-20000000 /rprojectnb2/kageproj/manta/results.epacts/test.11.10000001.20000000.epacts GT 0.01 1 10 1000000000 0.5 0 TRUE single.b.wald
Rscript /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install/share/EPACTS/epactsSingle.R --vanilla /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install /rprojectnb2/kageproj/manta/results.epacts/test.phe NULL /rprojectnb2/kageproj/manta/results.epacts/test.ind /rprojectnb2/kageproj/manta/results.epacts/manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz 11:20000001-30000000 /rprojectnb2/kageproj/manta/results.epacts/test.11.20000001.30000000.epacts GT 0.01 1 10 1000000000 0.5 0 TRUE single.b.wald
Rscript /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install/share/EPACTS/epactsSingle.R --vanilla /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install /rprojectnb2/kageproj/manta/results.epacts/test.phe NULL /rprojectnb2/kageproj/manta/results.epacts/test.ind /rprojectnb2/kageproj/manta/results.epacts/manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz 11:30000001-40000000 /rprojectnb2/kageproj/manta/results.epacts/test.11.30000001.40000000.epacts GT 0.01 1 10 1000000000 0.5 0 TRUE single.b.wald
Rscript /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install/share/EPACTS/epactsSingle.R --vanilla /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install /rprojectnb2/kageproj/manta/results.epacts/test.phe NULL /rprojectnb2/kageproj/manta/results.epacts/test.ind /rprojectnb2/kageproj/manta/results.epacts/manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz 11:40000001-50000000 /rprojectnb2/kageproj/manta/results.epacts/test.11.40000001.50000000.epacts GT 0.01 1 10 1000000000 0.5 0 TRUE single.b.wald
Rscript /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install/share/EPACTS/epactsSingle.R --vanilla /share/pkg.7/epacts/2020-01-21_gitfc9c3a4/install /rprojectnb2/kageproj/manta/results.epacts/test.phe NULL /rprojectnb2/kageproj/manta/results.epacts/test.ind /rprojectnb2/kageproj/manta/results.epacts/manta.gcad.duphold.pass.precise.norm.hwe.vcf.gz 11:50000001-60000000 /rprojectnb2/kageproj/manta/results.epacts/test.11.50000001.60000000.epacts GT 0.01 1 10 1000000000 0.5 0 TRUE single.b.wald
Loading required package: epactsR
Loading required package: epactsR
Loading required package: epactsR
Loading required package: epactsR
Loading required package: epactsR

WARNING -

WARNING -
Cannot parse region 11:10000001-20000000.. Returning emptyCannot parse region 11:50000001-60000000.. Returning empty

WARNING -
Cannot parse region 11:30000001-40000000.. Returning empty

WARNING -
Cannot parse region 11:20000001-30000000.. Returning empty

**Comment by jonathonl** [comment]
Yeah, that is unexpected behavior. I'm assuming you have multiple chromosomes in your VCF? As a workaround, can you try specifying a single chromosome with `--chr chr11` or `--region chr11:1-135086622`? If that still doesn't work, can you try the develop branch?
